src/helpers/utilities.js

```diff
@@ -147,9 +147,11 @@ export const handle = (promise) => {
 
  * @param {string} url - path to the resource
  * @param {object} params - query parameters
+ * @param {string} method – "GET" or "POST" for type of request
+ * @param {AbortSignal} signal – AbortController signal to abort the request
  * @return {Promise}
  */
-export function fetchData(url, params = {}, method = "GET") {
+export function fetchData(url, params = {}, method = "GET", signal) {
```

Comment:
Allows for aborting the current `fetch` request. Don't think this should affect any other tools but it's worth checking / keeping in mind.